cmd/algod/main.go

```diff
@@ -161,6 +161,10 @@ func main() {
 		if err != nil {
 			fmt.Fprintln(os.Stdout, "error loading telemetry config", err)
 		}
+		if os.IsPermission(err) {
+			fmt.Fprintf(os.Stdout, "Permission error on accessing telemetry config: %v", err)
```

Comment:
If we're going to exit the process here, than we should make it into a `os.Stderr`.

Reply:
I would also suggest that you'd add some more details here to explain the user which directories and/or files are attempted to be accessed, as I'm not sure if these would be included in the error message.